docs/theming.md

```diff
@@ -3,6 +3,8 @@
 The theming system is pretty straightforward. You can have your own skin in uw-frame with minimal effort. We highly
 encourage you to contribute back your theme to this project so you don't have to manage an independent fork of uw-frame.
 
+For a step-by-step exercise in setting up a theme, see [`skinning.md`](https://github.com/UW-Madison-DoIT/uw-frame/blob/master/docs/theming.md).
```

Comment:
I'd suggest relative links for within-repo links. That way when we change the hosting organization of the repository, we won't have to update these.  However, I think we use absolute links elsewhere as well, so we're going to have to pay the price anyway.